.eslintrc.cjs

```diff
@@ -11,4 +11,8 @@ module.exports = {
   env: {
     node: true,
   },
+  rules: {
+    "@typescript-eslint/no-unsafe-member-access": "off",
+    "@typescript-eslint/no-unsafe-assignment" : "off"
```

Comment:
Não consegui resolver o erro de acesso inseguro nas partes em que acesso o vetor de erros da resposta:
```
    expect(result?.data?.errors[0].message).to.equal(
      'Password is not strong enough: should be at least 6 characters long and have at least one letter and 1 digit.',
    );
```
`  expect(result?.data?.errors[0].message).to.equal('Email already in use.');`

Tentei ainda adicionar mais um operador de segurança (esta versão não é a commitada), mas o erro persistiu.
`expect(result?.data?.errors?.[0]?.message).to.equal( .... )`

O outro erro, de assignment, ocorre ao tentar atribuir o usuário retornado.
`const createdUser: User = createResult.data?.data?.createUser`

Se houver alguma sugestão para resolver estes erros sem precisar colocar as rules em off, eu agradeceria. 
* Já tentei também usar o tsconfig do 'Projetinho' disponibilizado, mas os mesmos erros persistiram.